src/monodroid/jni/util.h

```diff
@@ -15,9 +15,20 @@
 #define MONODROID_PATH_SEPARATOR "/"
 #endif
 
+#if WINDOWS
+typedef struct _stat monodroid_stat_t;
+#define monodroid_dir_t _WDIR
```

Comment:
That is *bizarre*. What was your typedef? Did you try using with `struct`?

    typedef struct _WDIR monodroid_dir_t;